packages/shared/lib/core/ledger/enums/ledger-app-name.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum LedgerAppName {
+    BOLOS = 'BOLOS',
```

Comment:
Forgot to mention last time - the enum variant names should be in `PascalCase` and the corresponding values in `camelCase`. I can't remember enough to know if the camelCase values will break anything, like locale usage or something.